src/multikey.rs

```diff
@@ -311,7 +300,7 @@ impl Multisig {
     pub fn to_legacy<W: Write>(&self, w: &mut W) -> Result<(), io::Error> {
         match self.0 {
             _Multisig::Ed25519(ref sig) => {
-                let data = base64::encode_config(&sig[..], base64::STANDARD);
+                let data = base64::encode_config(&sig.0[..], base64::STANDARD);
```

Comment:
The base64 encoding could be done by `Signature::as_base64`, `PublicKey::as_base64`, etc

Reply:
Done!